src/pages/registration/resource_type_tab/components/RegistrationTypesRow.tsx

```diff
@@ -0,0 +1,54 @@
+import { Typography, Box, Chip } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import FilterVintageIcon from '@mui/icons-material/FilterVintage';
+import { PublicationType } from '../../../../types/publicationFieldNames';
+import { PublicationInstanceType } from '../../../../types/registration.types';
+import { dataTestId } from '../../../../utils/dataTestIds';
+import { nviApplicableTypes } from '../../../../utils/registration-helpers';
+
+interface RegistrationTypesRowProps {
+  onChangeType: (type: PublicationInstanceType) => void;
+  mainType: PublicationType;
+  registrationType: RegistrationTypeElement[];
+  value: string;
+}
+
+export interface RegistrationTypeElement {
```

Comment:
Når vi la den her hit første gangen tenkte jeg det var midlertidig og at vi skulle flytte det til en mer "fornuftig" plass senere. Men nå klarer jeg ikke å se en plass som gir mening egentlig, så kanskje den bare skal få ligge her likevel... 🤔

Reply:
Når vi la det inn først tenkte jeg det var en generell ting som kunne ligge i en eller annen `types/****.d.ts` fil, men nå brukes den bare to plasser da, så sikkert like greit at den bare ligger i en av dem